docs/examples/form/custom-validation.vue

```diff
@@ -93,7 +93,7 @@ const submitForm = (formEl: FormInstance | undefined) => {
       console.log('submit!')
     } else {
       console.log('error submit!')
-      return false
+      return
```

Comment:
`return` can also be removed directly.

Reply:
Indeed, `return` does not make sense here.